app/controllers/api/irs_attempts_api_controller.rb

```diff
@@ -44,7 +44,14 @@ def authenticate_client
 
     def security_event_tokens
       return {} unless timestamp
-      @security_event_tokens ||= redis_client.read_events(timestamp: timestamp)
+
+      events = redis_client.read_events(timestamp: timestamp)
+      sets = {}
+      events.each_pair do |k, v|
+        key_id, jit = k.split(':')
+        sets[jit] = { key_id => v }
+      end
+      sets
```

Comment:
I'm not thrilled with this living in the controller, but we're getting a hash out of Redis and it's just doing some light data munging. 😐